README.rst

```diff
@@ -19,6 +21,8 @@ thus hopefully keeping the learning curve flat for the developer.
 
 .. |build status| image:: https://secure.travis-ci.org/Kotti/Kotti.png?branch=stable
 .. _build status: http://travis-ci.org/Kotti/Kotti
+.. |pypi| image:: https://pypip.in/version/zope.interface/badge.svg?style=flat
```

Comment:
I Think this needs to be https://pypip.in/version/Kotti/badge.svg?style=flat instead?